components/fieldgroups/experience-event/events/change-campaign-cadence.schema.json

```diff
@@ -29,10 +29,10 @@
               "title": "Change Campaign Cadence",
               "type": "object",
               "properties": {
-                "xdm:campaignID": {
```

Comment:
We do not allow breaking changes like removing a property from a schema. If there is a need to replace this, please deprecate the existing field and add another one.

Reply:
Got it. Will do that. Thanks.